nengo/builder/connection.py

```diff
@@ -94,12 +94,7 @@ def build_linear_system(model, conn, rng):
     eval_points = get_eval_points(model, conn, rng)
     ens = conn.pre_obj
     activities = get_activities(model.params[ens], ens, eval_points)
-    if np.count_nonzero(activities) == 0:
-        raise BuildError(
```

Comment:
It was non-trivial to hit this `BuildError` -- you basically had to be calling it with unrealistic inputs, since the builder will never get to this point if there were all zero inputs. So we opted to make it an `assert` so that we'd still know if we got here, but didn't have to make an extremely artificial test case.

Reply:
I've been able to hit it by having neurons with high intercepts, then using eval points that never cross those intercepts.